sdk/aot/azure-aot-graalvm-support/src/main/java/com/azure/aot/graalvm/support/implementation/features/AzureCoreFeature.java

```diff
@@ -2,14 +2,28 @@
 // Licensed under the MIT License.
 package com.azure.aot.graalvm.support.implementation.features;
 
+import com.azure.aot.graalvm.support.implementation.ClassReflectionAttributes;
 import com.azure.aot.graalvm.support.implementation.GraalVMFeature;
 import com.oracle.svm.core.annotate.AutomaticFeature;
 
+import java.util.Set;
+
+import static com.azure.aot.graalvm.support.implementation.ClassReflectionAttributes.createWithAllDeclared;
+import static com.azure.aot.graalvm.support.implementation.GraalVMFeatureUtils.setOf;
+
 @AutomaticFeature
 public class AzureCoreFeature implements GraalVMFeature {
 
     @Override
     public String getRootPackage() {
         return "com.azure.core";
     }
+
+    @Override
+    public Set<ClassReflectionAttributes> getReflectionClasses() {
+        return setOf(
+            createWithAllDeclared("com.azure.core.util.DateTimeRfc1123"),
+            createWithAllDeclared("com.azure.core.http.rest.StreamResponse")
+        );
+    }
```

Comment:
why do we need this?

Reply:
Problems found by testing require explicit declaration of these classes，it can be moved to `spring-cloud-azure-native-configuration`.